readme update for PR creation

xznichka opened the pull request.

xznichka committed:
readme update for PR creation

IvanZdanovich commented on README.md:
```diff
@@ -1,7 +1,5 @@
 # AutomationPracticeJS
```
Please do not use Russian language.

IvanZdanovich commented on README.md:
```diff
@@ -1,7 +1,5 @@
 # AutomationPracticeJS
 
```
please add exclusions for popular ide to .gitignore. Optimise exclusions for Cypress project

IvanZdanovich commented on README.md:
```diff
@@ -1,7 +1,5 @@
 # AutomationPracticeJS
 
-Это пример проекта для использования Cypress, инструмента для автоматического тестирования веб-приложений.
```
Do you really need following branches: develop, bugfix/*, hotfix/*

IvanZdanovich commented on README.md:
```diff
@@ -1,7 +1,5 @@
 # AutomationPracticeJS
 
-Это пример проекта для использования Cypress, инструмента для автоматического тестирования веб-приложений.
-
```
please add links to documentation files in README.md file

xznichka committed:
Fix comments

Changes of structure in git-strategy, readme, .gitignore, hotkeys
Change of language

IvanZdanovich commented on README.md:
Provide commands to README.md to run cypress in debug and test run modes.

xznichka committed:
commands for run cypress added
